docs/ann_types.md

```diff
@@ -46,81 +47,80 @@ Annotations with custom types:
+| `CCDS`               | `str`      | The CCDS identifer for this transcript, where applicable                                                                                    |                             |
+| `IND`                | `str`      | Individual name                                                                                                                             |                             |
+| `BIOTYPE`            | `str`      | Biotype of transcript or regulatory feature                                                                                                 |                             |
+| `APPRIS`             | `str`      | Annotates alternatively spliced transcripts as primary or alternate based on a range of computational methods. NB: not available for GRCh37 |                             |
+| `TSL`                | `str`      | Transcript support level. NB: not available for GRCh37                                                                                      |                             |
+| `GIVEN_REF`          | `str`      | Reference allele from input                                                                                                                 |                             |
+| `USED_REF`           | `str`      | Reference allele as used to get consequences                                                                                                |                             |
+| `AMBIGUITY`          | `str`      | IUPAC allele ambiguity code                                                                                                                 |                             |
+| `HGNC_ID`            | `str`      |                                                                                                                                             |                             |
+| `MANE`               | `str`      | Matched Annotation from NCBI and EMBL-EBI (MANE).                                                                                           |                             |
+| `MANE_SELECT`        | `str`      | Matched Annotation from NCBI and EMBL-EBI (MANE) canonical transcript.                                                                      |                             |
+| `MANE_PLUS_CLINICAL` | `str`      | MANE transcripts beyond MANE_SELECT that are clinically relevant.                                                                           |                             |
+| `GO`                 | `str`      | Gene ontology (GO) terms.                                                                                                                   |                             |
+| `miRNA`              | `str`      | Determines where in the secondary structure of a miRNA a variant falls                                                                      |                             |
```

Comment:
The thing is that because of the consequence removal, the table markdown changed (because column width changed).
Anyways, removing the type column is a good idea ;D